lib/screens/new_form_screens/new_form.dart

```diff
@@ -14,17 +14,20 @@ import 'package:firebase_storage/firebase_storage.dart';
 import 'package:project_eureka_flutter/services/email_auth.dart';
 import 'package:uuid/uuid.dart';
 import 'package:project_eureka_flutter/services/new_question_service.dart';
+import 'package:project_eureka_flutter/services/new_answer_service.dart';
 
 class NewForm extends StatefulWidget {
   final String categoryName;
   final bool isAnswer;
   final QuestionModel questionModel;
```

Comment:
You can remove this `questionModel` If you're just gonna use the String `questionId` instead. This is actually a better workflow.